src/com/fediroryshchuk/homework7/threads/queues/Consumer.java

```diff
@@ -0,0 +1,25 @@
+package com.fediroryshchuk.homework7.threads.queues;
+
+import java.util.concurrent.BlockingQueue;
+
+public class Consumer implements Runnable {
+    protected BlockingQueue queue;
+    private int count;
+
```

Comment:
Remove redundant blank line. There should be only one blank line between the fileds declaration and constructor.